src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <h1 hidden>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple iMac 27 inch Retina display"
+        class="card__image"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <span class="stars__reviews-text">Reviews: 5</span>
+      </div>
+      <p class="card__price">
+        <span>Price:</span>
+        <span class="card__price-number">$2,199</span>
+      </p>
+      <a
+        href="#"
+        class="card__link"
+        data-qa="hover"
+      >
+        BUY
+      </a>
```

Comment:
The 'BUY' text should be more descriptive for accessibility reasons. For example, 'Buy Apple iMac 27" Retina 5K' provides more context about the action.